css/Style.css

```diff
@@ -784,3 +784,88 @@ body {
   top: calc(50% - 40px / 2);
   border-radius: 8px;
 }
+
+#contact {
+  display: flex;
+  flex-direction: column;
+  order: 3;
+  width: 320px;
+  height: 779px;
+  background-image: url("../img/Contact\ me.png");
+  background-repeat: no-repeat;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
- [ ] Please stick to the Figma, kindly make sure you followed all the core requirements. take care of padding and margins. [check your design ](https://raw.githack.com/enoqueJonas/Portfolio/update-contact-form-style/index.html)and fix it, please. 👉[ figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

Reply:
@sajid-munawar  I can not access the exact values (width, height, padding, etc) on the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) I chose. It does not show the **form section** and its elements as it does on other templates. Can you please help me get the exact values?

Reply:
@enoqueJonas 
- Please check your `#works` id inside your CSS file and decrease padding or you can remove and then fix all parts of your design.